### PR TITLE
Add SimTrackerHit

### PR DESCRIPTION
**Pull request by vvolkl**
First Iteration: Taken from PLCIO, but with one long long int member for cellID instead of two ints, and without Extracode.

### CODE REVIEW

**Comment by vvolkl** [comment]
One question: For FCC this class was one of the bottlenecks and took a lot of diskspace. The MCParticle linked to the Hit also has infomation on "momentum", but not at the point where the Hit was created. Furthermore some Digitization algorithms depend on the angle of the particle path crossing the sensor, which this also provides, so I think this should be kept as is, no?

**Comment by gaede** [comment]
@vvolkl I had tried to make your PR compatible for rebasing but seem to have made a mistake.
I will simply create a new PR with Sim hits that we can include and then iterate on.

File: edm4hep.yaml [hunk 1]
```diff
@@ -5,61 +5,125 @@ options:
...
+  edm4hep::SimTrackerHit:
+    Description: "Simulated tracker hit"
+    Author : "F.Gaede, DESY"
+    Members:
+      - long long int cellID          //ID of the sensor that created this hit
+      - float EDep           //energy deposited in the hit [GeV].
+      - float time           //proper time of the hit in the lab frame in [ns].
+      - float pathLength     //path length of the particle in the sensitive material that resulted in this hit.
+      - int   quality        //quality bit flag.
+      - plcio::DoubleThree position //the hit position in [mm].
```
Comment (andresailer) [inline edm4hep.yaml]:
```suggestion
      - edm4hep::Vector3d position //the hit position in [mm].
```

File: edm4hep.yaml [hunk 2]
```diff
@@ -5,61 +5,125 @@ options:
...
+      - plcio::FloatThree momentum  //the 3-momentum of the particle at the hits position in [GeV]
```
Comment (andresailer) [inline edm4hep.yaml]:
```suggestion
      - edm4hep::Vector3f momentum  //the 3-momentum of the particle at the hits position in [GeV]
```

**Comment by vvolkl** [comment]
Thanks for the comments, should be ok now except that it needs some changes in podio (https://github.com/AIDASoft/podio/pull/60) but of course you can also create a new one @gaede.

**Comment by vvolkl** [comment]
Superseded by #11

**Pull request closed without merging**